RatScanner/View/MainWindow.xaml

```diff
@@ -11,6 +11,18 @@
         ResizeMode="CanMinimize"
         Height="400" Width="300">
 
+  <Window.Resources>
+    <Style TargetType="{x:Type Hyperlink}">
+      <Setter Property="TextBlock.FontSize" Value="14" />
+      <Setter Property="Foreground" Value="White" />
```

Comment:
Color should be `#EEEEEE` instead of `White`.

Reply:
Done